middlewares/validate-email-reset-password.js

```diff
@@ -16,7 +16,7 @@ const sendPasswordResetEmail = async (email, resetToken) => {
     from: user.email, //user email
     to: email,
     subject: "Restablecimiento de Contraseña",
-    html: `¡Hola! esperamos que te encuentres bien. Hemos recibido una solicitud para recuperar la contraseña de tu cuenta  Entendemos lo importante que es para ti acceder a tu cuenta de manera segura.  <br/>Haz clic en el siguiente enlace para restablecer tu contraseña: <br/>
+    html: `¡Hola! esperamos que te encuentres bien. <br/>Hemos recibido una solicitud para recuperar tu contraseña. Entendemos lo importante que es para ti acceder a tu cuenta de manera segura.  <br/>Haz clic en el siguiente enlace para restablecer tu contraseña: <br/>
```

Comment:
Sorry I forgot to let you the last suggestion.

```suggestion
    html: `¡Hola! esperamos que te encuentres bien. <br/>Hemos recibido una solicitud para recuperar tu contraseña. Entendemos lo importante que es para ti acceder a tu cuenta de manera segura.  <br/>Haz clic en el siguiente enlace para restablecerla: <br/>
```